cruise-control/src/main/java/com/linkedin/kafka/cruisecontrol/config/BrokerCapacityInfo.java

```diff
@@ -0,0 +1,40 @@
+/*
+ * Copyright 2018 LinkedIn Corp. Licensed under the BSD 2-Clause License (the "License"). See License in the project root for license information.
+ */
+
+package com.linkedin.kafka.cruisecontrol.config;
+
+import com.linkedin.kafka.cruisecontrol.common.Resource;
+import java.util.Map;
+
+
+public class BrokerCapacityInfo {
+  private final Map<Resource, Double> _capacity;
+  private final String _estimationInfo;
+
+  public BrokerCapacityInfo(Map<Resource, Double> capacity, String estimationInfo) {
```

Comment:
Minor: it might be good to have another constructor that only accepts the capacity, which sets the estimationInfo to "". That can be used more easily by the normal cases.